docs/guidelines/git-github.md

```diff
@@ -0,0 +1,306 @@
+# Git and Github
+
+This guideline contains some basic information on configuration of Git and user profiles on github.com. Our perspective would be security and privacy. The guideline is by no means exhaustive, it's more an introduction to basic config and the correlation between Git and github.com
+
+!!! tip "Git vs Github.com"
+
+    Git is a distributed version control system for tracking changes in source code, while GitHub is a platform that hosts Git repositories online. GitHub builds upon Git, offering a centralized place for developers to share and work on code together.
```

Comment:
There is something wrong with the format of the markdown every time this syntax is used:
![image](https://github.com/equinor/appsec/assets/88738856/a6153bb6-722b-470a-b07b-e9e9bdadb798)

Not sure if it would be noticeable on the live website? 🤔